source/dub/packagemanager.d

```diff
@@ -203,6 +203,55 @@ class PackageManager {
 		return pack;
 	}
 
+	/** For a given Git repository, returns the corresponding package.
+
+		Git repository is provided as its remote URL, the repository is cloned
+		and in the dependency speicfied commit is checked out.
+
+		If the target directory already exists, just returns the package
+		without cloning.
+
+		Params:
+			name = Package name
+			dependency = Dependency that contains the repository URL and a specific commit
+
+		Returns:
+			The package loaded from the given Git repository or null if the
+			package couldn't be loaded.
+	*/
+	Package clonePackage(string name, Dependency dependency)
```

Comment:
all the `git` references in the documentation should probably be replaced with just generic description that this method will download a repository if `repository` is set